stdlib/uri/0/mailto.rbs

```diff
@@ -4,5 +4,89 @@ module URI
   # RFC6068, the mailto URL scheme.
   #
   class MailTo < Generic
+    EMAIL_REGEXP: Regexp
```

Comment:
This constant is not documented in the official document.
https://docs.ruby-lang.org/en/3.3/URI/MailTo.html

But it's well used for verifying the format of email addresses.
For example, the Rails Guide mentions this constant in implementing validators on Rails.
https://guides.rubyonrails.org/active_record_validations.html#custom-validators